src/main/lang/de.json

```diff
@@ -6,6 +6,13 @@
         "short": "AP",
         "use": "AP-Kosten"
       },
+      "actions": {
+        "attack": {
+          "aim": "Zielen",
+          "calledShot": "Angekündigter Schuss",
```

Comment:
This doesn't seem like a great translation. I keep coming back to "Gezielter Schuss", but that could be confused with aiming in general.

Reply:
That's exactly why I picked this one instead. It's clunky, but it seems like the only option to keep things unambiguous